packages/landingpage/app/[lang]/_components/layout/header/desktop/header.desktop.tsx

```diff
@@ -1,29 +1,24 @@
 import ElementsLogo from '@assets/elements.svg';
 import Image from 'next/image';
 import Link from 'next/link';
-import useTranslation from '../../../../utils/hooks/useTranslation';
-import { useRouter } from 'next/router';
 import styles from './header.desktop.module.scss';
 import LocaleSwitcher from './localeSwitcher';
 import Navbar from './navbar';
-import VersionSelect from '../../../shared/versionSelect';
+import VersionSelect from '../versionSelect';
+import { usePathname } from 'next/navigation';
 
 export default function HeaderDesktop() {
-  const { locale } = useTranslation();
-  const router = useRouter();
-  const isLibraryPage = router.pathname.endsWith('/library/components');
-
+  const pathName = usePathname();
+  const isLibraryPage = pathName?.endsWith('/library/components');
   return (
     <header className={styles.header}>
       <div className={styles.headerInner}>
         <div className={styles.logo}>
           <Link
             className={styles.logoLink}
             href={{
-              pathname: '/[lang]',
-              query: { lang: locale },
+              pathname: '.',
             }}
```

Comment:
Bug: If I'm in `http://localhost:4600/en/library/components?element=docs-welcome--docs`, clicking the logo takes me to the starting page of the storybook instead of the home page.